### ISSUE TITLE
RoboUiController.injectString does not respect case

### ISSUE THREAD

**Issue by acrab** [robolectric/robolectric#4634]
### Description
When testing apps which require user input, a common behaviour is to type some text into them, and then verify that that text is propagated correctly through the app.
When the provided text contains capital letters, that text does not appear to be propagated through robolectric as expected, causing the test to fail.

### Steps to Reproduce
1. Create a layout containing an EditText
2. Create an activity which inflates this layout
3. Create a test which launches the activity
4. Perform a typeText action on the EditText, providing a string containing capital letters
5. Check the text of the EditText, and note that it does not contain capitals.

In its simplest form, this test is as follows:
```
@Test
fun test_text()
{
    launchActivity<MainActivity>()
    onView(withId(R.id.editText)).perform(ViewActions.typeText("Mixed Case Text"))
    onView(withId(R.id.editText)).check(matches(withText("Mixed Case Text")))
}
```

### Robolectric & Android Version
Robolectric: 4.0.2
compileSdkVersion 28

### Link to a public git repo demonstrating the problem:

[https://github.com/acrab/text-test](https://github.com/acrab/text-test)

Repository contains an Android Studio project
Example test is in ExampleUnitTest.kt
Running this test shows the problem

**Comment by jongerrish** [comment]
This may have been fixed in 4.1 - mind giving it a try?

**Comment by acrab** [comment]
Yep, that's fixed it! Thanks :+1: 

I've pushed a new commit to the example repository showing this.
